packages/vscode-extension/src/commands/SetupAccount.ts

```diff
@@ -256,6 +256,14 @@ export default class SetupAccount extends BaseAction {
 	}
 
 	private async getPrivateKeyFilePath() {
+		if (!await window.showQuickPick(['Select private key file'], {
```

Comment:
Please use translation service for string values.

Reply:
Maybe we could reuse a modified version of `SETUP_ACCOUNT_CREATE_M2M_SELECT_PRIVATE_KEY_FILE`

Reply:
I forgot about. Pushed changes.